packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -141,12 +141,12 @@ export class SummaryManager implements IDisposable {
         // enforce connectedState.clientId === clientElection.electedClientId. But once we're Running, we should
```

Comment:
This comment needs to be updated to clearly describe why the "parentShouldNotSummarize" condition is needed. Can you please create an issue for this? I meant to change this comment but it slips my mind.

Reply:
Also, these stop reasons need to be renamed. I don't think the names convey what they mean. You can probably add this to the same issue.

Reply:
#10105